base/setup/reactos/lang/ru-RU.rc

```diff
@@ -76,19 +76,18 @@ BEGIN
     PUSHBUTTON "&Отставить", IDCANCEL, 87, 68, 47, 15, WS_VISIBLE | WS_TABSTOP
 END
 
-IDD_BOOTOPTIONS DIALOGEX 0, 0, 305, 116
+IDD_ADVINSTOPTS DIALOGEX 0, 0, 305, 135
 STYLE DS_SHELLFONT | WS_VISIBLE | WS_CAPTION
-CAPTION "Дополнительные параметры установки"
+CAPTION "Advanced Installation Options"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    CONTROL "Основной каталог", IDC_STATIC, "Button", BS_GROUPBOX, 4, 1, 298, 30
-    EDITTEXT IDC_PATH, 10, 12, 283, 14, WS_VISIBLE
-    CONTROL "Установка загрузчика", IDC_STATIC, "Button", BS_GROUPBOX, 4, 36, 298, 52
-    CONTROL "Установить загрузчик на диск (MBR и VBR)", IDC_INSTFREELDR, "Button", BS_AUTORADIOBUTTON | WS_TABSTOP, 10, 49, 278, 11
-    CONTROL "Установить загрузчик на диск (только VBR)", IDC_INSTVBRONLY, "Button", BS_AUTORADIOBUTTON | WS_TABSTOP, 10, 59, 278, 11
-    CONTROL "Не устанавливать загрузчик", IDC_NOINSTFREELDR, "Button", BS_AUTORADIOBUTTON | WS_TABSTOP | WS_DISABLED, 10, 69, 278, 11
-    PUSHBUTTON "&OK", IDOK, 184, 94, 50, 15, WS_TABSTOP | WS_VISIBLE
-    PUSHBUTTON "&Отмена", IDCANCEL, 244, 94, 50, 15, WS_TABSTOP | WS_VISIBLE
+    LTEXT "Choose the &directory where you want to install ReactOS:", IDC_STATIC, 7, 9, 291, 10
+    EDITTEXT IDC_PATH, 7, 23, 291, 13, WS_VISIBLE
+    GROUPBOX "Установка загрузчика", IDC_STATIC, 7, 45, 291, 60
+    LTEXT "Select the location where the FreeLoader\nbootloader should be installed.\n\nBy default, it is installed on the system partition of the boot disk (and either in the Master or the Volume Boot Record for BIOS-based computers).", IDC_STATIC, 13, 57, 279, 44
+    COMBOBOX IDC_INSTFREELDR, 150, 58, 142, 45, WS_VSCROLL | WS_TABSTOP | CBS_DROPDOWNLIST
+    PUSHBUTTON "Есть", IDOK, 193, 113, 50, 14, WS_TABSTOP | WS_VISIBLE
```

Comment:
```suggestion
    PUSHBUTTON "ОК", IDOK, 193, 113, 50, 14, WS_TABSTOP | WS_VISIBLE
```
Or it's intentional?

Reply:
I think I've copy-pasted it from somewhere else.